metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -48,14 +48,14 @@ case class ScalafixProvider(
   // Warms up the Scalafix instance so that the first organize imports request responds faster.
   def load(): Unit = {
     if (!Testing.isEnabled) {
+      val tmp = workspace
```

Comment:
What was actually happening here? How come moving it out of the `try` fixes it?

Reply:
Not 100 percent sure 😅 it seems there was a race, but how I have no idea 😑

Reply:
I just added additional checks to see if evaluation indeed finishes successfully and it seems it does. There might be some magic behind the finally block.